spec/index.html

```diff
@@ -506,6 +516,7 @@ <h3>Quoted Literals</h3>
         show:218 rdfs:label "That Seventies Show"^^<http://www.w3.org/2001/XMLSchema#string> . # same as above
         show:218 rdfs:label "That Seventies Show" .                                            # same again
         show:218 show:localName "That Seventies Show"@en .                 # literal with a language tag
+        show:218 show:localName "That Seventies Show"@en-ltr .             # literal with a language tag and base direction
         show:218 show:localName 'Cette Série des Années Soixante-dix'@fr . # literal delimited by single quote
         show:218 show:localName "Cette Série des Années Septante"@fr-be .  # literal with a region subtag
```

Comment:
Might also want to add an example with both regional sub-tag and base direction, like `@fr-be--ltr`?